apps/web/pages/api/cron/bookingReminder.ts

```diff
@@ -29,6 +29,8 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         createdAt: {
           lte: dayjs().add(-interval, "minutes").toDate(),
         },
+        // Only send reminders if the event hasn't finished
+        endTime: { gte: new Date() },
```

Comment:
Wouldn’t be better if we changed this to startTime? And add little interval time? What I understand its that this reminder its for accept pending bookings, right?
I think we don't wanna send email if the booking has started and not finish